src/plugins/regions.ts

```diff
@@ -70,8 +73,10 @@ class SingleRegion extends EventEmitter<RegionEvents> {
   public content?: HTMLElement
   public minLength = 0
   public maxLength = Infinity
+  public channelId: number
+  public wavesurfer: WaveSurfer
 
-  constructor(params: RegionParams, private totalDuration: number) {
+  constructor(params: RegionParams, private totalDuration: number, wavesurfer: WaveSurfer) {
```

Comment:
I intentionally didn't pass the wavesurfer instance to the Region class to limit its responsibilities.
The only place where you're using it is to get the numberOfChannels, so I'd suggest to pass that instead.

Reply:
I see it, thanks for explaining! I just pushed this change